channels/1ds-post-js/src/PostChannel.ts

```diff
@@ -1187,4 +1246,13 @@ export class PostChannel extends BaseTelemetryPlugin implements IChannelControls
     public _clearBackOff() {
         // @DynamicProtoStub - DO NOT add any code as this will be removed during packaging
     }
+
+    /**
+     * Get Offline Serializer support
+     * @returns internal Offline Serializer object
+     */
+    public getOfflineSupport() {
```

Comment:
nit: this should specify that it returns the interface so that the typescript definition is reported correctly.